docs/guides/device-gateway/sdk.md

```diff
@@ -279,7 +279,7 @@ the request:
 		"protocol": "http",
 		"forwards_to": "localhost:8001",
 		"domain": "device123.sitea.configurable-domain.com",
-		"policy": "{\"inbound\": [], \"outbound\": [{\"expressions\": [], \"name\": \"Add headers to requests\", \"actions\": [{\"type\": \"add-headers\", \"config\": {\"headers\": {\"is-ngrok\": \"0000000000\"}}}]}]}"
+		"policy": "{\"on_http_request\": [], \"on_http_response\": [{\"expressions\": [], \"name\": \"Add headers to requests\", \"actions\": [{\"type\": \"add-headers\", \"config\": {\"headers\": {\"is-ngrok\": \"0000000000\"}}}]}]}"
```

Comment:
Think this whole guide should have some updates from `policy` > `traffic_policy`

Reply:
yep, missed this.